utils/lib/rr_graph/graph.py

```diff
@@ -578,7 +542,28 @@ def _add_pin(self, pin):
 
 
 class BlockType(MostlyReadOnly):
-    '''For <block_type> nodes'''
+    """
+    For <block_type> nodes
+
+    Attributes
+    ----------
+    graph : Graph
+
+    id : int
+
+    name : str
+        Name of the block type.
+
+    size : Size
+        
+
+    pin_classes : tuple of PinClass
+        
+
+    pin_index : int
+        ????
```

Comment:
Why ??? Do you need clarification on something? Or just didn't finish writing?

Reply:
Fixed.